test/runner/main.go

```diff
@@ -135,6 +132,12 @@ func runTestCaseNative(testBin string, tc gtest.TestCase, t *testing.T) {
 		cmd.SysProcAttr.Cloneflags |= unix.CLONE_NEWNET
 	}
 
+	if cmd.SysProcAttr.Cloneflags&unix.CLONE_NEWNET != 0 {
+		// setup_container takes in its target argv as positional arguments.
```

Comment:
I probably don't understand this too well, why do we do this step only when creating a new network namespace?

Reply:
Possibly update comment.

Reply:
setup_container initializes the loopback interface.

Reply:
I will submit this change in a separate pr.